undead-discord.css

```diff
@@ -251,6 +251,25 @@
   line-height: 25px !important;
 }
 
+.headerDiscriminator-3fLlCR, .chat .new-messages-bar button:last-child {
+    color: var(--fg-color);
+}
+
+.container-3NvGrL, .header-1tSljs {
+    height: 57px;
+}
+
+.container-3lnMWU {
+    border-bottom: none;
+}
+
+.private-channels .search-bar {
+    color: var(--bg-color) !important;
+    border-color: var(--bg-color);
+    height: 54px;
+    background: var(--bg-color) !important;
+}
+
```

Comment:
Would be nice to move all the stuff out to the variables that already exist. Stuff like the `corol: var(--fg-color);` already has its own block. Same with the `background` and `border-color`. Height is probably the only thing that's can be its own block.